jupyter-declarativewidgets-incorporation/jupyter-declarativewidgets-extension-incorporation.md

```diff
@@ -0,0 +1,181 @@
+
+#### Have an active developer community that offers a sustainable model for future development.
+
+The project has contributions from 13 developers on GitHub and has been under active development over the past 10 months. We foresee future development of the extension taking the following primary forms:
+
+1. Fixes for reported issues
+2. Compatibility updates for new versions of Jupyter Notebook (e.g., 4.3, 4.4, ...)
+3. Contributed enhancements to the existing extension within the scope of the project
+4. Enhancements to ease the authoring experience
+5. Further work on exploring DataFrames
+
+All items can be handled by current or future maintainers.
+
+#### Have an active user community.
+
+The `jupyter-incubator/declarativewidgets` repository has 55 stars and 18 watchers on GitHub. We have received questions, bug reports, and enhancement requests on GitHub and the Google Group.
+
+#### Use solid software engineering with documentation and tests hosted with appropriate technologies.
+
+The extension has comprehensive suite of unit tests. These include unit tests for all supported languages, plus a unit tests for the web code. It also has a suite of system tests that use Selenium to drive the Notebook UI validate key features. These tests are run on a matrix that includes different versions of Python, Jupyter, ipywidgets and kernel languages.
+
+The project used Travis CI. All PRs are validated against a single browser vendor. On a PR merge, all tests are ran using SauceLabs on a variety of browser vendors.
+
+Deployment of releases to Pypi is automated using Git tags and Travis CI.
+
+Installation documentation is available in the README. Usage and API documentation is available off the [project site](https://jupyter-incubator.github.io/declarativewidgets/) or embedded in the Notebook using the `urth-help` element. Architectural details are available in the [project's wiki](https://github.com/jupyter-incubator/declarativewidgets/wiki).
+
+#### Demonstrate continued growth and development.
+
+See "Have an active developer community that offers a sustainable model for future development."
+
+#### Integrate well with other official Subprojects.
+
+The declarative extensions have several examples of interoperation with `ipywidgets`. It also includes a dedicate element (`urth-core-ipywidget`) that can embed an ipywidget within a declarative template.
+
+#### Be developed according to the Jupyter governance and contribution model.
+
+The declarativewidgets extension is in the Jupyter Incubator, and under the Jupyter governance and contribution model since its inception.
+
+#### Have a well-defined scope.
+
+The purpose of the declarativewidgets extension is to provide a language neutral approach to creating interactive notebooks. It is not the intention of this project to become a widget suite, rather, enable incorporating any web-component into a notebook and connect it to the code and data in the kernel.
```

Comment:
...  but rather, to enable incorporating any web-component into a notebook and connecting it to the code and data in the kernel.